### DISCUSSION TITLE
upgrade instructions for 4.1.3 tagged release: important instructions have been lost

### DISCUSSION THREAD

**Discussion by jdkruzr** [mastodon/mastodon#25783] [General]
Sorry, I can't find where the readmes are for the tagged releases themselves or I'd just submit a PR, but:

Both Docker and non-Docker:

Precompile the assets:
Non-Docker: RAILS_ENV=production bundle exec rails assets:precompile
Docker: The assets are already precompiled during the build step
Restart all Mastodon processes

^-- the bit about what to do for non-Docker releases is not here under Upgrade Notes: https://github.com/mastodon/mastodon/releases/tag/v4.1.3

**Comment by renchap** [comment]
Assets precompilation is not needed when upgrading from 4.1.2, as we took care of not including any CSS/JS changes in this version.

**Reply by jdkruzr** [reply]
ok. I was coming from 4.0.something, so it was needed in my case.

**Reply by renchap** [reply]
We mentioned in the release notes that if you were upgrading from another version than 4.1.2, you needed to read the other release notes as well, and apply their upgrade steps.
